operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -4074,6 +4074,10 @@ tokengenJob:
   extraEnvFrom: []
   annotations: {}
   initContainers: []
+  # -- Controls whether to store the generated admin token in a Kubernetes secret
+  storeTokenInSecret: true
+  # -- Name of the secret to store the admin token. If not specified, defaults to "<release-name>-admin-token"
+  adminTokenSecret: ""
```

Comment:
can we have the default string here instead? it's self-documenting and it won't go out of date

Reply:
fixed